packer/build.pkr.hcl

```diff
@@ -7,7 +7,8 @@ build {
     host_alias = "${var.ansible_host_alias}"
     playbook_file = "${var.playbook_file_path}"
     extra_arguments  = [
-      "-e", "aws_region=${var.aws_region}"
+      "-e", "aws_region=${var.aws_region}",
+      "-vv"
```

Comment:
can remove the -vv if we dont need verbose output anymore.

Reply:
Now removed:

 extra_arguments  = [
      "-e", "aws_region=${var.aws_region}"
    ]